site/content/internal/writing-a-blog-post.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Writing a Blog Post
+date: 2019-02-06T14:28:35-05:00
+section: internal
+weight: 105
+---
+
+Went to a conference recently? Worked on something cool? Got something else Mattermost related you want to post about? Follow these steps to write a blog post:
+
+1. Clone https://github.com/mattermost/mattermost-developer-documentation 
+2. Create a new .md file in the [/site/content/blog/](https://github.com/mattermost/mattermost-developer-documentation/tree/master/site/content/blog) folder
```

Comment:
Is there anything built into the blog software to automate this? I've looked (< 5 minutes) a few times, but never found anything, but it seems odd that they require you to, for example, figure out the date and time.

Reply:
There is but you need to install hugo first then it's `hugo new` or something like that